augly/video/transforms.py

```diff
@@ -405,7 +462,12 @@ def apply_transform(
 
         @returns: the path to the augmented video
         """
-        return F.brightness(video_path, output_path, self.level, metadata=metadata)
+        return F.brightness(
+                video_path,
+                output_path,
+                level=self.chosen_value,
+                metadata=metadata,
```

Comment:
Be careful with indentation
```suggestion
            video_path,
            output_path,
            level=self.chosen_value,
            metadata=metadata,
```